devops/index.md

```diff
@@ -11,7 +11,4 @@ image:
 ---
 
 
-{% for post in site.categories.devops %}
```

Comment:
@harshadyeola Dont delete anything please
All you need to add your new content.